### PR TITLE
Update some content

### PR DESCRIPTION
**Pull request by tiansworld**

### CODE REVIEW

File: convert_glossary.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+# This is my first python script that does some real work :-)
+# Written by Tian Shixiong <tiansworld@fedoraproject.org>
+# License: GPLv3+
+
+#try:
+#import tkinter as tk
+
+#except ImportError as ie:
+#    print('Import tkinter failed:',ie,'GUI read file won\'t work.',
+#          'Use open function instead' + '\n')
```
Comment (alick) [inline convert_glossary.py:14]: Will the commented out lines be useful? Can you just delete them?

File: convert_glossary.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+#! /usr/bin/env python3.3
```
Comment (alick) [inline convert_glossary.py:1]: python3. (Seems you're using a rather old python version.)

File: convert_glossary.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+filename = input('Please input the filename to convert, include its path if you need, but don\'t add quotes: ')
+file_to_read = open(filename,'r')
+savename = input('Please input the filename that you want to save, include its path if you need, but don\'t add quotes: ')
+file_to_save = open(savename,'w')
+#else:
+#file_to_read = tk.filedialog.askopenfile('r')
+#file_to_save = tk.filedialog.asksaveasfile('w')
+    
+
+class ConvertFile:
```
Comment (alick) [inline convert_glossary.py:24]: FileConverter (class names are usually nouns; func names are usually verb_actions)

File: convert_glossary.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+            # First add '!' at the begining of every English term
+            beginchar = '!'
+            
+            for char in s:
+                if char != ",":
+                    beginchar += char
+                else:
+                    beginchar += '\n' + '|'
+            beginchar += '\n' + '|-' + '\n'
```
Comment (alick) [inline convert_glossary.py:51]: There is a csv module in standard library for line parsing: https://docs.python.org/3.8/library/csv.html?highlight=csv

**Review by alick — CHANGES_REQUESTED** [review]
A few comments

**Comment by tiansworld** [comment]
On 14/5/2021 12:00 pm, Alick Zhao wrote:
> @alick commented on this pull request.
> 
> 
> 
>> +#try:
> +#import tkinter as tk
> +
> +#except ImportError as ie:
> +#    print('Import tkinter failed:',ie,'GUI read file won\'t work.',
> +#          'Use open function instead' + '\n')
> 
> Will the commented out lines be useful? Can you just delete them?
> 

Sorry, I should have updated or just removed that python script. I wrote
it many years ago after I completed learning my python online course on
Coursera.
I haven't had any more chance to practice and improve my python writing
skill after that, so the script is left there without any update made.

When I update the other files yesterday, I forgot to handle it.

I would be appreciate if you can improve it. Honestly, I couldn't
remember 80% part of what I've learnt and written.(What a pity!)


-- 
Regards,

Tiansworld
Fedora Project Contributor

**Comment by tiansworld** [comment]
On 14/5/2021 12:03 pm, Alick Zhao wrote:
> @alick commented on this pull request.
> 
> 
> 
>> +#try:
> +#import tkinter as tk
> +
> +#except ImportError as ie:
> +#    print('Import tkinter failed:',ie,'GUI read file won\'t work.',
> +#          'Use open function instead' + '\n')
> +filename = input('Please input the filename to convert, include its path if you need, but don\'t add quotes: ')
> +file_to_read = open(filename,'r')
> +savename = input('Please input the filename that you want to save, include its path if you need, but don\'t add quotes: ')
> +file_to_save = open(savename,'w')
> +#else:
> +#file_to_read = tk.filedialog.askopenfile('r')
> +#file_to_save = tk.filedialog.asksaveasfile('w')
> +    
> +

I think I was trying to add a GUI part then, but with no succeed.

> +class ConvertFile:
> 
> FileConverter (class names are usually nouns; func names are usually verb_actions)
> 

Great thanks for this tip.


-- 
Regards,

Tiansworld
Fedora Project Contributor

**Comment by gbraad** [comment]
> I wrote it many years ago

it might be helpful to just squash these changes? It's a lot of noise and the actual date it was written, history, etc  does not make much sense anymore.

File: convert_glossary.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        """ (ConvertFile) -> Nonetype
+        Save converted string to new file file_to_save
+        """
+        self.file_to_write.write(self.newcontent)
+
+    def close(self):
+        self.file_to_read.close()
+        self.file_to_write.close()
+
+##    def __str__(self):
```
Comment (gbraad) [inline convert_glossary.py:66]: ditto... comments

Comment (tiansworld) [reply convert_glossary.py:66]: Thanks a lot.
But seems I made it dirtier due to careless rebase in last week. (I've merged my squashed changes(in another branch) to this branch. So the changes were applied twice, one squashed and one not.)
Please ignore this PR.

**Pull request closed without merging**